### PR TITLE
TW-1500 Postbot LC doc changes for Dec 15

### PR DESCRIPTION
**Pull request by jkonrath-postman**

### CODE REVIEW

File: src/pages/docs/billing/billing.md [hunk 1]
```diff
@@ -231,12 +231,16 @@ Update the number of Flow Editor seats, monitoring blocks, mock server blocks, a
 
 ### Purchasing add-on products
 
-You can purchase Flow Editor and Postbot seats by navigating to your [billing dashboard](https://go.postman.co/billing/overview) and selecting **Edit plan** on the right.
+> The Postbot add-on is currently available for Free, Basic and Professional plans.
+
+If you are on Basic or Professional plans, you can purchase Flow Editor and Postbot seats by navigating to your [billing dashboard](https://go.postman.co/billing/overview) and selecting **Edit plan** on the right.
```
Comment (nerminamiller-postman) [inline src/pages/docs/billing/billing.md:236]:
```suggestion
If you are on a Basic or Professional plan, you can purchase Flow Editor and Postbot seats by navigating to your [billing dashboard](https://go.postman.co/billing/overview) and selecting **Edit plan** on the right.
```

File: src/pages/docs/billing/billing.md [hunk 2]
```diff
@@ -231,12 +231,16 @@ Update the number of Flow Editor seats, monitoring blocks, mock server blocks, a
...
 
 Select **Modify Plan** under the plan type for your team. Update the number of Flow Editor and Postbot seats in the **Add-on products** section, then select **Next**. Select **Confirm Changes** to purchase more seats.
 
 You can manage which team members consume Postbot seats from the [product access dashboard](https://go.postman.co/settings/team/products-access). Learn how to [manage your team's access to Postbot](/docs/administration/managing-your-team/managing-your-team/#manage-your-teams-access-to-postbot).
 
+If you are a free user, you can purchase Postbot seats by selecting **Upgrade** in the Postman header and navigating to the bundle section to add Postbot. Update the number of Postbot seats and select from Annual or Monthly payment options, then select **Next**. Enter your billing information and payment method, then select **Confirm**.
```
Comment (nerminamiller-postman) [inline src/pages/docs/billing/billing.md:242]:
```suggestion
If you are a Free plan user, you can purchase Postbot seats by selecting **Upgrade** in the Postman header and navigating to the bundle section to add Postbot. Update the number of Postbot seats and select from Annual or Monthly payment options, then select **Next**. Enter your billing information and payment method, then select **Confirm**.
```

**Review by nerminamiller-postman — APPROVED** [review]
LGTM! Just a couple of small suggestions.

**Review by MatthewPinsker-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/getting-started/basics/about-postbot.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ contextual_links:
     url: "https://blog.postman.com/generative-ai-and-the-impact-on-apis-and-software-development/"
 ---
 
-> Postbot is free to try on the Free plan. Postbot is available as an add-on to the [Basic and Professional plans](https://www.postman.com/pricing/). Postbot isn't currently available in the Enterprise plan.
+> Postbot is available as an add-on to the [Free, Basic, and Professional plans](https://www.postman.com/pricing/). Postbot isn't currently available in the Enterprise plan.
```
Comment (taehoshino) [inline src/pages/docs/getting-started/basics/about-postbot.md:20]: "the Enterprise plan" or should this be "Enterprise plans"? I think we usually use "Enterprise plans" (plural)

**Review by taehoshino — APPROVED** [review]
LGTM mostly. Please add Shivam Sharma to review as well.

**Pull request merged**